test/aave-stop-loss-multiply.ts

```diff
@@ -16,15 +16,15 @@ import { setBalance } from '@nomicfoundation/hardhat-network-helpers'
 import { TriggerGroupType } from '@oasisdex/automation'
 import { expect } from 'chai'
 
-describe('AaveStoplLossCommand-Multiply', async () => {
+describe.skip('AaveStoplLossCommand-Multiply', async () => { //TODO:this test need fixing, you can not hardcode proxy, you should test latest implementation of guard and factory
```

Comment:
this is fixed in #169
we do the same for all maker tests - we use an existing position, we dont create a new one.